handlers/incident.go

```diff
@@ -83,6 +150,29 @@ func apiCreateIncidentHandler(w http.ResponseWriter, r *http.Request) {
 	sendJsonAction(incident, "create", w, r)
 }
 
+func apiUpdateIncidentHandler(w http.ResponseWriter, r *http.Request) {
+	incident, _, err := findIncident(r)
```

Comment:
A bit hard to understand how different `incident` and `incidentVar` are

Reply:
changed the name to updatedIncident